eng/Versions.props

```diff
@@ -88,6 +88,7 @@
     <MicrosoftDotNetCodeAnalysisVersion>8.0.0-beta.23411.1</MicrosoftDotNetCodeAnalysisVersion>
     <MicrosoftDotNetGenAPIVersion>8.0.0-beta.23411.1</MicrosoftDotNetGenAPIVersion>
     <MicrosoftDotNetGenFacadesVersion>8.0.0-beta.23411.1</MicrosoftDotNetGenFacadesVersion>
+    <MicrosoftDotNetXUnitAssertVersion>8.0.0-beta.23411.1</MicrosoftDotNetXUnitAssertVersion>
```

Comment:
You probably also want a Version.Details.xml entry to keep the version up-to-date.

Reply:
Right! Nice catch, thanks